pkg/volume/cinder/cinder_block_test.go

```diff
@@ -70,7 +70,7 @@ func TestGetVolumeSpecFromGlobalMapPath(t *testing.T) {
 	}
 	block := v1.PersistentVolumeBlock
 	specMode := spec.PersistentVolume.Spec.VolumeMode
-	if &specMode == nil {
+	if specMode == nil {
 		t.Errorf("Invalid volumeMode from GlobalMapPath spec: %v expected: %v", &specMode, block)
```

Comment:
Could you also change this to `t.Fatalf()`? Otherwise the text execution wouldn't end here and line 76 will panic.

Reply:
Thanks for the review.